src/EA.Weee.RequestHandlers.Tests.DataAccess/Admin/FindMatchingUsersDataAccessTests.cs

```diff
@@ -381,7 +381,7 @@ public async void GetOrganisationUsers_WithOrganisationNameFilter_ReturnsMultipl
 
                 var dataAccess = new FindMatchingUsersDataAccess(dbWrapper.WeeeContext);
 
-                var filter = new UserFilter { OrganisationName = "as" };
```

Comment:
This looks like its trying to test a partial in word search, maybe change the Organisation name in the filter to be "ast" ...  or find out what its failing in your local environment